### DISCUSSION TITLE
What to do to change the menu font of Qt/Linux version?

### DISCUSSION THREAD

**Discussion by PNBRQK** [orbitalquark/textadept#509] [Q&A]
I've learned the UI font of the GTK version can be changed by creating GTK CSSes, but what about the Qt version?
I built the app from its latest source cloned from github, and installed it to a non-conventional location (such as `~/my_programs/bin`).

**Comment by orbitalquark** [comment]
I don't actually know the answer to this. Presumably your Linux desktop provides a theme application/service for Qt apps, and that should change fonts and sizes (as well as colors, etc.).

**Comment by YahYahDev** [comment]
i use qt5ct and qt6ct for customizing the qt theming, here is a link to the project! https://sourceforge.net/projects/qt5ct/

if you need info on how to configure it i found alot of info on the arch wiki thats useful.
